src/components/MessageInput/hooks/useAttachments.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Did you mean to leave this disable in?

Reply:
I'm not sure how this was added, I'll remove.